packages/commonwealth/client/scripts/views/pages/CommunityManagement/Contests/ContestsList/ContestsList.tsx

```diff
@@ -97,8 +95,10 @@ const ContestsList = ({
                 topics={contest.topics}
                 // @ts-expect-error <StrictNullChecks/>
                 score={score}
-                finishDate={moment(end_time).toISOString()}
-                isActive={!contest.cancelled && !hasEnded}
+                decimals={contest.decimals}
+                ticker={contest.ticker}
+                finishDate={end_time ? moment(end_time).toISOString() : ''}
```

Comment:
lets localize this date -> `moment().utc().local()`

Reply:
we don't do this anywhere else, why should we do it here?

Reply:
Actually, we want to provide local datetime to users in case of time-bound operations. There was a related bug fix here: https://github.com/hicommonwealth/commonwealth/issues/8036 and the product team decided to use localized dates everywhere (will send content via slack). In the bug fix, I only localized thread/comment datetime but eventually we should localize all other datetimes to local user time.

Reply:
makes sense, but I am not showing this date explicitly to the user. I am just calculating countdown based on this. Thus, I would probably leave it as it is and then if we have a ticket that covers dates modification, we can take a closer look